content/en/docs/tasks/inject-data-application/define-command-argument-container.md

```diff
@@ -31,7 +31,7 @@ the `args` field in the configuration file. The command and arguments that
 you define cannot be changed after the Pod is created.
 
 The command and arguments that you define in the configuration file
-override the default command and arguments provided by the container image.
+override the default command and arguments provided by the container image.（which are CMD and ENTRYPOINT）
```

Comment:
This is not correct. The fields are `command` and `args`, as called out in the previous paragraph.

https://github.com/kubernetes/website/blob/main/content/en/examples/pods/commands.yaml#L11-L12

If you want to point out their equivalents in `Dockerfile` syntax, then I think that needs to be called out in a little more detail than this vague statement.

Reply:
We shouldn't assume people use Docker (or a `Dockerfile`) to build their container images.

Reply:
Yeah! Thanks! I understand that image is not all from Docker. 
Too many people use docker to build image and I just want to help someone searching -  pod command will override dockerfile CMD.
I will try to add scene description.

Reply:
@stmcginnis @sftim Sincerely, are there any corresponding usages of Docker's CMD in other OCI standards? 

https://github.com/opencontainers/image-spec/blob/main/config.md
OCI Image Format define Cmd and Entrypoint, I think people say CMD more in image context and command in k8s/pods.

thanks for  replying

Reply:
> Sincerely, are there any corresponding usages of Docker's CMD in other OCI standards?

Yes, in the image spec that you linked to. For a container image in OCI format, the field names are `Cmd` and `Entrypoint`.

**However**, in this issue we're describing the Pod API, where it's called `command` and `entrypoint`.